pgmpy/tests/test_models/test_BayesianModel.py

```diff
@@ -132,6 +132,19 @@ def test_moral_graph_with_edge_present_over_parents(self):
             self.assertTrue(edge in [('a', 'b'), ('c', 'b'), ('d', 'a'), ('d', 'b'), ('d', 'e')] or
                             (edge[1], edge[0]) in [('a', 'b'), ('c', 'b'), ('d', 'a'), ('d', 'b'), ('d', 'e')])
 
+    def test_get_ancestors_of_success(self):
+        G = BayesianModel([('d', 'g'), ('g', 'l'), ('i', 'g'), ('i', 'l')])
```

Comment:
@kris-singh Since you are using this graph in 2 methods put this in the `__init__` method.